src/Mono.Android/Mono.Android.csproj

```diff
@@ -30,6 +30,11 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
 
+   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
```

Comment:
Indentation here is "wonky".

Where should netcoreapp3.1 output go?

Rando thought that I'm trying in PR #4376: place netcoreapp3.1 output into `$(topdir)\bin\Debug-netcoreapp3.1`.

Better idea -- unimplemented because I don't know the details -- is for files to be included in the .NET 5 support package to follow the appropriate directory structure of the .NET 5 support package.  The problem is that I don't know what the directory structure should be...

Reply:
For the targeting pack you can look at the rough destination paths here:
https://github.com/xamarin/xamarin-android/blob/net5-preview/src/Xamarin.Android.App/Xamarin.Android.App.Ref.csproj#L33

I like the idea of better mirroring this output, though there's a couple of questions/concerns. To do this we'll need to only build the latest stable version of `Mono.Android.dll` when targeting `netstandard2.1`, and not every supporting version. We also need ultimately need Mono.Android.dll in both targeting and runtime packs, so we'll need to account for that in the future as well. 

I'm not sure how much of this should be considered for this PR. I took a quick look at attempting to only build the latest `Mono.Android.dll` question and didn't find a great quick solution to that, but we may want to dedicate time to that aspect now at least so that we don't bloat our build times unnecessarily.

Reply:
Isn't netstandard2.1 too restrictive? You won't be able to use any newer APIs.

Reply:
I've updated the second target to `netcoreapp3.1`.

Reply:
I fixed the indentation, and changed the output path to use the `TargetFramework` for now, now that only the latest stable version will be built when targeting `netcoreapp3.1`.

Reply:
> To do this we'll need to only build the latest stable version of `Mono.Android.dll` when targeting `netstandard2.1`, and not every supported version.

The intention is that net5-android will redistribute *only* API-R/API-30 `Mono.Android.dll`, and not one for every previous `$(TargetFrameworkVersion)` value.

Reply:
The latest changes ensure that we only build the `$(AndroidLatestStableFrameworkVersion )` version of `Mono.Android.dll` when targeting `netcoreapp3.1`.